pycytominer/cyto_utils/collate_cmd.py

```diff
@@ -0,0 +1,92 @@
+if __name__ == "__main__":
+    import argparse
+    from pycytominer.cyto_utils.collate import collate
```

Comment:
I think convention usually has these imported before the if name main statement IIRC

Reply:
Yup, sorry, that's an artifact of this being moved out from the main file